frontend-react/playwright.config.ts

```diff
@@ -81,35 +81,29 @@ export default defineConfig<TestOptions>({
     },
 
     projects: [
-        /* Test setup (ex: authenticated sessions) */
         { name: "setup", testMatch: /\w+\.setup\.ts$/ },
+        // We have a suite of tests that are ONLY checking links so to
+        // save bandwidth, we only need to utilize a single browser
+        {
+            name: "chromium-specific",
+            use: { browserName: "chromium" },
+            testMatch: "spec/chromium-only/*.spec.ts",
+        },
         {
             name: "chromium",
             use: { browserName: "chromium" },
-            dependencies: ["setup"],
+            testMatch: "spec/*.spec.ts",
```

Comment:
The dependencies property SHOULD NOT be removed.

```suggestion
            dependencies: ["setup"],
            testMatch: "spec/*.spec.ts",
```